index.html

```diff
@@ -1,295 +1,342 @@
-                  <p class="role">Back End Dev</p>
-                  <span class="seprator-circle"></span>
-                  <p class="year">2015</p>
-                </div>
-              </div>
-              <p class="project-details">
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required.
-              </p>
-              <ul class="tags">
-                <li class="tag">html</li>
-                <li class="tag">css</li>
-                <li class="tag">javaScript</li>
+        </div>
+        <a href="#" class="hamburger-menu">
+          <div class="line"></div>
+          <div class="line"></div>
+          <div class="line"></div>
+        </a>
+      </nav>
+    </header>
+    <!-- headline section-->
+    <section class="headline">
+      <div class="title">
+        <h1>
+          I’m Ritika <br />
+          Glad to see you!
+        </h1>
+        <p class="details">
+          I’m a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <header class="links-text">Let's connect</header>
+        <ul class="icons-section">
+          <li>
+            <a href="https://twitter.com/Ritikarawat22">
+              <img src="./images/tw.png" alt="twitter"/></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/rawatritika/">
+              <img src="./images/Linkedin.png" alt="Linkedin"/></a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/medium.png" alt="medium"/></a>
+          </li>
+          <li>
+            <a href="https://github.com/ritikarawat220">
+              <img src="./images/Github.png" alt="github"
+            /></a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./images/v.png" alt="angellist" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="project_section">
+      <!-- first project -->
+      <div class="first_project">
+        <div class="Epxlore_image">
+           <img class="Nature" src="./desktop/nature.jpg" alt="Nature"  />
+           <img class="Tonic" src="./portfolio/tonic.png" alt="Tonic" />
+        </div>
+        <div class="tonic_section">
+          <h2>Tonic</h2>
+          <div class="flex_list">
+                  <ul class="list">
+                    <h5>CANOPY</h5>
+                    <li class="list_items">Back End Dev</li>
+                    <li class="list_items">2015</li>
+                  </ul>
+                  <p class="project_description">
+                     A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+                  </p>
+                  <div class="tags">
```

Comment:
As requested by the previous reviewer, kindly ensure that the margins, paddings, alignments and all other spacings of the project section follows that of the figma design. There are still some variations between your project and that of the figma. Kindly check and fix them. 


![Screenshot from 2023-01-20 13-06-43](https://user-images.githubusercontent.com/65233860/213702827-e0fec317-199f-40dd-a194-be435ce67edd.png)

![Screenshot from 2023-01-20 13-06-55](https://user-images.githubusercontent.com/65233860/213702845-59f0ba43-98e4-4307-bb13-620df0d26517.png)